.github/workflows/prerelease.yml

```diff
@@ -24,6 +24,10 @@ jobs:
           python-version: "3.9"
       - name: "Install Python Packages"
         run: "pip install poetry"
+      - name: "Install Docs Requirements"
+        run: "pip install -r docs/requirements.txt"
+      - name: "Build Documentation"
+        run: "mkdocs build --no-directory-urls"
```

Comment:
It would be good to document this as part of the backup (manual) build-and-publish steps in `nautobot/docs/development/release-checklist.md` as well.

Reply:
Good idea, I will add these changes then.